.web-docs/components/builder/tart/README.md

```diff
@@ -0,0 +1,337 @@
+- `create_grace_time` (duration string | ex: "1h5m2s") — Time to wait after finishing the installation process. Can be used to work around the issue when Virtualization.Framework's installation process is still running in the background for some time after `tart create` had already finished.
+- `disk_size_gb` — Disk size in GB to use for the new VM. Overrides `tart create` default value when using `from_ipsw` and `from_iso` and VM settings when using `vm_base_name`.
+- `display` (string) — VM display resolution in a format of `<width>x<height>` (e.g. `1200x800`) to use for the new VM. Overrides `tart create` default value when using `from_ipsw` and `from_iso` and VM settings when using `vm_base_name`.
+- `from_ipsw` (string) - Location of an IPSW file to initialize a macOS virtual machine from. Can be either an absolute path to a file on disk, URL to fetch a remote file or `latest`. Mutually exclusive with `from_iso` and `vm_base_name`.
+- `from_iso` (list(string)) - Location of the ISO files to initialize a Linux virtual machine from. All values should represent an absolute path to a file on disk. Mutually exclusive with `from_ipsw` and `vm_base_name`.
+- `headless` (bool) - Whether to show graphics interface of a VM. Useful for debugging the `boot_command`.
+- `memory_gb` (number) - Amount of unified memory in GB to use for the new VM. Overrides `tart create` default value when using `from_ipsw` and `from_iso` and VM settings when using `vm_base_name`.
+- `recovery` (bool) — Whether to boot the VM in recovery mode. Useful for disabling the System Integrity Protection automatically for the already created VMs.
+- `rosetta` (string) - Whether to enable Rosetta support of a Linux guest VM. Useful for running non-arm64 binaries in the guest VM. A common used value is `rosetta`, for further details and explanation run `tart run --help`.
+- `run_extra_args` (list(string)) - Extra arguments to pass to `tart run` command. For example, you can enable bridged networking by specifying `--net-bridged=en0`.
+- `ip_extra_args` (list(string)) - Extra arguments to pass to `tart ip` command. For example, you can use a different resolver in case of bridged network by specifying `--resolver=arp`.
+- `vm_base_name` (string) - The name of the VM to be used for the initial cloning. Can be either a local VM or a remote VM that will be pulled from a registry. Mutually exclusive with `from_ipsw` and `from_iso`.
+
+### SSH connection configuration
+
+- `ssh_username` (string) - Username to use for the communication over SSH to run provision steps.
+- `ssh_password` (string) - Password to use for the communication over SSH to run provision steps.
+
+## HTTP server configuration
+
+<!-- Code generated from the comments of the HTTPConfig struct in multistep/commonsteps/http_config.go; DO NOT EDIT MANUALLY -->
```

Comment:
@nywilken is there a way to include these properly, without copy-and-pasting the generated code?

So that upstream changes will be propagated without any action on our side.

Reply:
Oh, got it, they're auto-generated from the old `tart.mdx`.